--- libmspack-0.7.1alpha/Makefile.am.orig	2018-08-09 10:29:14.000000000 +0200
+++ libmspack-0.7.1alpha/Makefile.am	2018-10-12 19:31:46.296437020 +0200
@@ -60,39 +60,39 @@
 			mspack/readbits.h mspack/readhuff.h
 libmschmd_la_LDFLAGS =	-export-symbols-regex '^mspack_'
 
-examples_cabd_memory_SOURCES =	examples/cabd_memory.c libmscabd.la
+examples_cabd_memory_SOURCES =	examples/cabd_memory.c
 examples_cabd_memory_LDADD =	libmscabd.la
-examples_multifh_SOURCES =	examples/multifh.c libmscabd.la
+examples_multifh_SOURCES =	examples/multifh.c
 examples_multifh_LDADD =	libmscabd.la
 
-src_cabrip_SOURCES =		src/cabrip.c libmspack.la
+src_cabrip_SOURCES =		src/cabrip.c
 src_cabrip_LDADD =		libmspack.la
-src_chmextract_SOURCES =	src/chmextract.c src/error.h libmspack.la
+src_chmextract_SOURCES =	src/chmextract.c src/error.h
 src_chmextract_LDADD =		libmspack.la
-src_msexpand_SOURCES =		src/msexpand.c src/error.h libmspack.la
+src_msexpand_SOURCES =		src/msexpand.c src/error.h
 src_msexpand_LDADD =		libmspack.la
-src_oabextract_SOURCES =	src/oabextract.c src/error.h libmspack.la
+src_oabextract_SOURCES =	src/oabextract.c src/error.h
 src_oabextract_LDADD =		libmspack.la
 
 test_cabd_md5_SOURCES =		test/cabd_md5.c test/md5.c test/md5.h \
-				test/md5_fh.h test/error.h libmscabd.la
+				test/md5_fh.h test/error.h
 test_cabd_md5_LDADD =		libmscabd.la
-test_cabd_test_SOURCES=		test/cabd_test.c libmscabd.la
+test_cabd_test_SOURCES=		test/cabd_test.c
 test_cabd_test_LDADD =		libmscabd.la
-test_chmd_find_SOURCES =	test/chmd_find.c test/error.h libmschmd.la
+test_chmd_find_SOURCES =	test/chmd_find.c test/error.h
 test_chmd_find_LDADD =		libmschmd.la
 test_chmd_md5_SOURCES =		test/chmd_md5.c test/md5.c test/md5.h \
-				test/md5_fh.h test/error.h libmschmd.la
+				test/md5_fh.h test/error.h
 test_chmd_md5_LDADD =		libmschmd.la
 test_chmd_order_SOURCES =	test/chmd_order.c test/md5.c test/md5.h \
-				test/md5_fh.h test/error.h libmschmd.la
+				test/md5_fh.h test/error.h
 test_chmd_order_LDADD =		libmschmd.la
-test_chmd_test_SOURCES =	test/chmd_test.c libmschmd.la
+test_chmd_test_SOURCES =	test/chmd_test.c
 test_chmd_test_LDADD =		libmschmd.la
 test_chmd_test_DEPENDENCIES = 	libmschmd.la test/test_files/chmd/cve-2015-4467-reset-interval-zero.chm
-test_chminfo_SOURCES =		test/chminfo.c libmschmd.la
+test_chminfo_SOURCES =		test/chminfo.c
 test_chminfo_LDADD =		libmschmd.la
-test_kwajd_test_SOURCES =	test/kwajd_test.c libmspack.la
+test_kwajd_test_SOURCES =	test/kwajd_test.c
 test_kwajd_test_LDADD = 	libmspack.la
 
 test/test_files/chmd/cve-2015-4467-reset-interval-zero.chm: test/test_files/chmd/cve-2015-4467-reset-interval-zero.chm.LZXC-is-lzxc
