--- libmspack-0.9.1alpha/Makefile.am.orig	2018-12-29 11:03:21.843727561 +0100
+++ libmspack-0.9.1alpha/Makefile.am	2018-12-29 11:08:59.620390371 +0100
@@ -12,6 +12,7 @@
 AM_CPPFLAGS =           -I$(srcdir)/mspack -I$(srcdir)/test
 
 include_HEADERS =       mspack/mspack.h
+pkginclude_HEADERS =	mspack/cab.h mspack/chm.h mspack/crc32.h mspack/des.h mspack/hlp.h mspack/kwaj.h mspack/lit.h mspack/lzss.h mspack/lzx.h mspack/mszip.h mspack/oab.h mspack/qtm.h mspack/readbits.h mspack/readhuff.h mspack/sha.h mspack/system.h mspack/szdd.h
 lib_LTLIBRARIES =       libmspack.la
 pkgconfig_DATA =        libmspack.pc
 
